libvuln/jsonblob/jsonblob.go

```diff
@@ -73,8 +87,15 @@ func (l *Loader) Next() bool {
 			l.next.Fingerprint = l.de.Fingerprint
 			l.next.Date = l.de.Date
 		}
-		l.next.Vuln = append(l.next.Vuln, l.de.Vuln)
-		l.de.Vuln = nil // Needed to ensure the Decoder allocates new backing memory.
+		i := len(vs)
+		vs = append(vs, claircore.Vulnerability{})
+		if err := json.Unmarshal(l.de.Vuln, &vs[i]); err != nil {
+			l.err = err
+			return false
+		}
+		l.next.Vuln = append(l.next.Vuln, &vs[i])
```

Comment:
Does this mean that we will have a memory footprint the size of all the vulnerabilities in the `Entry{}`?

Reply:
On load, yes. There's no way around that in the current interface.